README.rst

```diff
@@ -84,12 +86,27 @@ Example settings
 Advanced customisation
 ----------------------
 
-Getting IP
-~~~~~~~~~~
+Getting IP Address
+~~~~~~~~~~~~~~~~~~
+
+By default, ``BasicAuthIPWhitelistMiddleware`` uses ``request.META["REMOTE_ADDR"]``
+as the client's IP, which corresponds to the IP address connecting to Django.
+If you have a reverse proxy (eg ``nginx`` in front), this will result in the IP address of
+``nginx``, not the client.
+
```

Comment:
**Suggestion:** My opinion is: let's not introduce the default that is just a bad default. Let's completely disable the IP check unless the function is configured.